pkg/model/azuremodel/vmscaleset.go

```diff
@@ -207,7 +207,7 @@ func getStorageProfile(spec *kops.InstanceGroupSpec) (*compute.VirtualMachineSca
 	if spec.RootVolume != nil && spec.RootVolume.Type != nil {
 		storageAccountType = compute.StorageAccountTypes(*spec.RootVolume.Type)
 	} else {
-		storageAccountType = compute.StorageAccountTypesPremiumLRS
+		storageAccountType = compute.StorageAccountTypesStandardSSDLRS
```

Comment:
If this change is intentional then it may be worthy of a release note

Reply:
Azure is still Alpha, so not sure it's worth the effort.